packages/shared/src/components/icons/Clear/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react';
+import Icon, { IconProps } from '../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+export const ClearIcon = (props: IconProps): ReactElement => (
+  <Icon {...props} IconPrimary={OutlinedIcon} IconSecondary={FilledIcon} />
+);
+
+export default ClearIcon;
```

Comment:
Can you maybe remove the default export.
Just changed all the icons to only have named ones :)

Reply:
Yeah, I have to rebase.